packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -119,18 +120,31 @@ export class Gauge extends PureComponent<Props> {
   }
 
   render() {
-    const { height, width } = this.props;
+    const { height, width, value } = this.props;
 
     return (
       <div
         style={{
           height: `${Math.min(height, width * 1.3)}px`,
           width: `${Math.min(width, height * 1.3)}px`,
-          top: '10px',
           margin: 'auto',
+          marginTop: '-8px',
         }}
         ref={element => (this.canvasElement = element)}
-      />
+      >
+        {value.title && (
```

Comment:
Thinking of integrating this into the jquery flot gauge rendering as I want the placement a bit higher which will need to push the value up a bit. But we can start with this.

Reply:
👍 i was aiming for easy